### ISSUE TITLE
Allways Transparent.

### ISSUE THREAD

**Issue by dmmg83** [rotorgames/Rg.Plugins.Popup#195]
Hello, Thanks for your work @rotorgames , I have this issue:

When I run a popup page on android, it's truning content transparent. No matters what i do, it's always transparent.

Version 1.0.4
Compiling with Android 8.0

Xalm:
`<pages:PopupPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:pages="clr-namespace:Rg.Plugins.Popup.Pages;assembly=Rg.Plugins.Popup"
             x:Class="AppDemo..Demo">
<StackLayout VerticalOptions="Center" HorizontalOptions="Center" Padding="0" Margin="0">
        <Frame Margin="30" Padding="10">
            <Label HorizontalTextAlignment="Center" FontSize="Large" Text="Hello popup!"></Label>
        </Frame>
    </StackLayout>
</pages:PopupPage>`

C#:
`await Navigation.PushPopupAsync(new Demo());`

**Comment by brianjtools** [comment]
Try setting the BackgroundColor of the StackLayout white or some other color otherwise it will be transparent.  Set the WidthRequest and HeightRequest to whatever size you want the popup to be.

**Comment by dmmg83** [comment]
Thank you @brianjtools. I already tried that, nothing work.

**Comment by ipon85** [comment]
Hello, got same problem. Is there any working solution?

**Comment by rotorgames** [comment]
@dmmg83 Hi. Thank you for report. Can you create a simple project that has this problem?

**Comment by simpletasks** [comment]
@dmmg83  in code snippet you wrote `await Navigation.PushPopupAsync(new Demo());`
I do not see that method in 1.0.4 version. I use `PopupNavigation.PushAsync()`.
In my basic usage only with PushAsync() method I get transparent background. If I use xamarin default Navigation class then popup background is opaque.
Added your xaml code as new page in existing project and "Hello popup" is opaque. Rest of page is transparent.
Compile for Android 8.0
popup plugin 1.0.4
Run on Android 6.0 and 7.1.1.

**Comment by rotorgames** [comment]
@simpletasks In the first try to update to the last pre release if it doesn't help you please check that system animations in Android are enabled.

**Comment by simpletasks** [comment]
Hi. Made new project to test with v1.0.4 and with 1.1.3.73-pre. Both versions works as expected with sample code from @dmmg83 comment. 
```xml
<pages:PopupPage
    x:Class="XamApp1.Page11"
    xmlns="http://xamarin.com/schemas/2014/forms"
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    xmlns:pages="clr-namespace:Rg.Plugins.Popup.Pages;assembly=Rg.Plugins.Popup">
    <StackLayout
        Margin="0"
        Padding="0"
        BackgroundColor="Yellow"
        HorizontalOptions="Center"
        VerticalOptions="Center">
        <Frame Margin="30" Padding="10">
            <Label
                FontSize="Large"
                HorizontalTextAlignment="Center"
                Text="Hello popup!" />
        </Frame>
    </StackLayout>
</pages:PopupPage>
```

On mainPage added buton to push popup page to stack:
```c#
private async void Nav_button(object sender, EventArgs e)
        {
            await PopupNavigation.PushAsync(new Page11());
        }
```
also tested with
```c#
using Rg.Plugins.Popup.Extensions;
. . . 
private async void Nav_button(object sender, EventArgs e)
        {
            await Navigation.PushPopupAsync(new Page11());
        }
```
Around button on MainPage added labels just to fill MainPage to see where popup page is transparent. When button is pressed yellow box is shown with white label "Hello popup". Space around yellow box is transparent as expected.
As tested there is no problem with this plugin as reported.
Tested for Android. Compile API 19-26. Run on Android 6.0 and 7.1.1.

**Comment by rotorgames** [comment]
I think that I have fixed it in the beta release v1.1.3.121-beta. You can test it from the development NuGet: https://ci.appveyor.com/nuget/rg-plugins-popup-p6l5elugev99

P.S. Not afraid to reopen this issue if you have this problem again.

Thanks
